TelegramBot/Program.cs

```diff
@@ -11,7 +11,8 @@
 
 public class Program
 {
-    static ITelegramBotClient bot = new TelegramBotClient("5842668948:AAG8kaYQH4xHEk6tykiOmIFFwMyRxdWpdAA");
+    static string botToken = Environment.GetEnvironmentVariable("TELEGRAM_BOT_API_TOKEN");
```

Comment:
but  in this case with my api i getting a null variable

Reply:
Yeah, you are getting null value, because you haven't set environment variable in the cli where you are running your dotnet application from

Reply:
In PowerShell: `$env:TELEGRAM_BOT_API_TOKEN="yourapitoken"`
In normal command prompt: `set TELEGRAM_BOT_API_TOKEN="yourapitoken"`

Reply:
Or you can just [set it globally](https://docs.oracle.com/en/database/oracle/machine-learning/oml4r/1.5.1/oread/creating-and-modifying-environment-variables-on-windows.html#GUID-DD6F9982-60D5-48F6-8270-A27EC53807D0)

Reply:
i appreciate and thank you mate that it is very helpful in the field of security. i already fixed and updated my repo